src/pages/register.tsx

```diff
@@ -141,7 +141,7 @@ const Register = ({ inputChecks, errors, regKey, csrfToken }: RegisterProps & Cu
                                             <input
                                                 className="govuk-checkboxes__input"
                                                 id="checkbox-user-research"
-                                                name="checkbox-user-research"
+                                                name="checkboxUserResearch"
                                                 type="checkbox"
                                                 value="checkUserResearch"
```

Comment:
We could change this here so that `name="contactable"` and `value="yes"`. We could then remove any handling/reassignment in register.ts